core/whistle_apps-1.0.0/src/whapps_speech.erl

```diff
@@ -290,6 +354,37 @@ create_response(_Engine, {'error', _R}) ->
 create_response(_Engine, {'ibrowse_req_id', ReqID}) ->
     lager:debug("speech file streaming as ~p", [ReqID]),
     {'ok', ReqID};
+create_response(<<"voicefabric">> = _Engine, {'ok', "200", Headers, Content}) ->
+    [lager:debug("hdr: ~p", [H]) || H <- Headers],
+    lager:debug("converting media"),
+    RawFile = tmp_file_name(<<"raw">>),
+    WavFile = tmp_file_name(<<"wav">>),
+    _ = file:write_file(RawFile, Content),
+    From = "raw -r 8000 -e signed-integer -b 16",
```

Comment:
Take a look at `wh_media_util:normalize_media/3`. It does what you're doing here, but via a port_command, avoiding creating temporary files to cleanup.